Add/Modify new keywords for Ver.4.1 to 4.5

yasuking0304 opened the pull request:
## Add/Modified new keywords for Ver.4.1 to 4.5

I referred to here and picked up the missing basic keywords.

https://genshin-impact.fandom.com/wiki

- added version 4.1-4.5 characters
- added version 4.4 drops-boss
- added version 4.4 drops
- added version 4.4 enemies
- added version 4.4 items
- added version 4.4 living beings
- added version 4.4 locations
- added version 4.4 objects
- added version 4.3-4.5 weapons
- added version 4.3-4.5 y-events
- modified japanese notes(delete tag's playable character)
- modified japanese/simplified notes
- added/modified pronunciationJa, and delete notes check comment

### issue

Closes #187

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/xicri/genshin-langdata?pullRequest=188) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/xicri/genshin-langdata?pullRequest=188) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/xicri/genshin-langdata?pullRequest=188) it.</sub>

xicri commented:
@yasuking0304 Thanks, I will review it!
In the meantime, can you sign the Contributor License Agreement as the bot asked in the above comment?

@Bill-Haku @SleepyAsh0191 Can you review Chinese part of this PR if you have time?

Bill-Haku commented on dataset/dictionary/locations.json5:
```diff
@@ -682,6 +682,89 @@
+    tags: [ "liyue", "location" ],
+  },
+  {
+    // The following place name translation is incorrect, so check it on PS5 <a href=\"https://genshin-impact.fandom.com/wiki/Mt._Xuanlian\">
+    en: "Mt. Xuanlian",
+    ja: "悬练山",
+    zhCN: "灵濛山",
```
```suggestion
    en: "Mt. Xuanlian",
    ja: "悬练山",
    zhCN: "悬练山",
```

Yes, it's incorrect. But I'm just wondering why check it on **PS5**?

yasuking0304 replied:
Sorry. It's a copy and paste mistake
Plogrem
```
    en: "Mt. Xuanlian",
    ja: "悬练山",
    zhCN: "灵濛山",
```
after
```
    en: "Mt. Xuanlian",
    ja: "懸練山",
    zhCN: "悬练山",
```

Bill-Haku reviewed (changes requested):
Thanks for your contribution @yasuking0304 !

SleepyAsh0191 commented on dataset/dictionary/y-events.json5:
```diff
@@ -186,6 +186,151 @@
+    zhCN: "铓锋锦间裁",
+    pronunciationJa: "ほうじんにしきをたつ",
+    tags: [ "event" ],
+    notes: "v4.5 期間限定イベント",
```
Not event name, just code name only

yasuking0304 replied:
Attached is the revised document.
problem
```
    notes: "v4.5 期間限定イベント",
```
after
```
    notes: "v4.5 バージョンサブタイトル",
```

SleepyAsh0191 reviewed (approved):
There are no problems in Chinese part.

SleepyAsh0191 commented:
> There are no problems in Chinese part.

Oh, I forgot the part of bill commented xD

xicri reviewed (changes requested):
@yasuking0304 I've added some review comments.

Here are some other requests:

- If there are no official source for `pronunciationJa`, can you add or keep `// TODO Need Check` comment? While it is technically meaningless currently, I will add a property to show if the Japanese pronunciation is reliable one or just our guess.
- Can you fix the CI errors? I guess there are some JSON5 syntax errors and lint errors.

Bill-Haku commented on dataset/dictionary/characters.json5:
```diff
@@ -1096,7 +1104,22 @@
+    pronunciationJa: "かんうん",
+    "tags": [ "liyue", "character-main" ],
+    notes: "留雲借風真君/三眼五顯仙人",
+    notesZh: "留云借风真君/三眼五显仙人",
```
```suggestion
    notesZh: "留云借风真君的俗名",
```

Sure. I think @yasuking0304 knows how to edit it for other items.

xicri replied:
@Bill-Haku Thanks!

Bill-Haku commented on dataset/dictionary/characters.json5:
```diff
@@ -1107,6 +1130,8 @@
     zhCN: "萍姥姥",
     tags: [ "liyue", "character-sub", "sereniteapot" ],
+    notes: "歌塵浪市真君/三眼五顯仙人",
+    notesZh: "歌尘浪市真君/三眼五显仙人",
```
```suggestion
    notesZh: "雅号为“歌尘浪市真君”",
```

but in fact both 尊称 and 雅号 can be used here in Chinese and maybe 尊称 is a more commonly used word in Chinese. It depends on you @yasuking0304 .

yasuking0304 commented on dataset/dictionary/characters.json5:
```diff
@@ -1096,7 +1104,22 @@
     pronunciationJa: "がんしん",
     variants: {
       en: [ "Geo God", "God of Geo" ],
-    },
```
Sorry. It's a correction mistake. Fixed.

xicri replied:
I guess [the CI error currently raised](https://github.com/xicri/genshin-langdata/actions/runs/8446217592/job/23134664258?pr=188) is caused by this.

yasuking0304 commented on dataset/dictionary/items.json5:
```diff
@@ -795,7 +803,7 @@
     ja: "萃凝晶",
     zhCN: "萃凝晶",
-    pronunciationJa: "すいぎょうしょう", // TODO Need Check
+    pronunciationJa: "すいぎょうしょう",
```
Sorry. As you pointed out, I put it back.

yasuking0304 commented on dataset/dictionary/weapons.json5:
```diff
@@ -1378,7 +1378,7 @@
     ja: "純水流華",
     zhCN: "纯水流华",
-    pronunciationJa: "じゅんすいるか", // TODO Need Confirmation
+    pronunciationJa: "じゅんすいりゅうか",
```
Sorry. As you pointed out, I put it back.

yasuking0304 commented on dataset/dictionary/weapons.json5:
```diff
@@ -1462,21 +1462,61 @@
     ja: "久遠流転の大典",
     zhCN: "万世流涌大典",
-    pronunciationJa: "くおんるてんのだいてん", // TODO Need Check
+    pronunciationJa: "くおんるてんのたいてん",
```
Sorry. As you pointed out, I put it back.

yasuking0304 commented on dataset/dictionary/weapons.json5:
```diff
@@ -1462,21 +1462,61 @@
     en: "Cashflow Supervision",
     ja: "凛流の監視者",
     zhCN: "金流监督",
+    "pronunciationJa": "りるのかんししゃ",
```
As you pointed out, I added 'TODO Need Check'

yasuking0304 commented on dataset/dictionary/items.json5:
```diff
@@ -178,8 +178,16 @@
     en: "Crimson Agate",
     ja: "緋紅玉髄",
     zhCN: "绯红玉髓",
+    pronunciationJa: "ひこうぎょくずい",
```
As you pointed out, I added 'TODO Need Check'

yasuking0304 commented:
Sorry. The lint check was not done and the build was failing, so I took measures.

- Responding to lint indications
- add comma, "}",
- fixed invalid white space
- Added support for Windows for lint check (package.json -> 'lint_windows')

xicri commented on dataset/dictionary/living-beings.json5:
```diff
@@ -165,6 +165,53 @@
+    notesZh: "一只生活在遗珑埠以北海湾的巨龟",
+  },
+  {
+    en: "Jadestone Turtle",
```
Maybe you forget to update this to correct English translate when you copied-and-pasted.

yasuking0304 replied:
Sorry. Duplicate English: Jadestone Turtle
Plogrem
```
    en: "Jadestone Turtle",
    ja: "秋羽鴨",
    zhCN: "秋绒鸭",
```
after
https://genshin-impact.fandom.com/wiki/Velvetfall_Duck
```
    en: "Velvetfall Duck",
    ja: "秋羽鴨",
    zhCN: "秋绒鸭",
```

yasuking0304 commented:
Sorry. Duplicate English: Jadestone Turtle
Plogrem
```
    en: "Jadestone Turtle",
    ja: "秋羽鴨",
    zhCN: "秋绒鸭",
```
after
https://genshin-impact.fandom.com/wiki/Velvetfall_Duck
```
    en: "Velvetfall Duck",
    ja: "秋羽鴨",
    zhCN: "秋绒鸭",
```

xicri commented on dataset/dictionary/characters.json5:
```diff
@@ -1650,6 +1676,93 @@
+    pronunciationJa: "こうしょう",
+    tags: [ "liyue", "character-sub" ],
+    notes: "削月築陽真君/三眼五顯仙人",
+    notesZh: "削月筑阳真君/三眼五显仙人"
```
@yasuking0304 Can you apply the same change as https://github.com/xicri/genshin-langdata/pull/188#discussion_r1540380408?

xicri commented on dataset/dictionary/characters.json5:
```diff
@@ -1650,6 +1676,93 @@
+    pronunciationJa: "せっしゃく",
+    tags: [ "liyue", "character-sub" ],
+    notes: "理水畳山真君/三眼五顯仙人",
+    notesZh: "理水叠山真君/三眼五显仙人"
```
@yasuking0304 Can you apply the same change as https://github.com/xicri/genshin-langdata/pull/188#discussion_r1540380408?

xicri commented on dataset/dictionary/y-events.json5:
```diff
@@ -186,6 +186,136 @@
+    zhCN: "弘辩的溢神效应",
+    pronunciationJa: "ゆうべんなフェーイズさよう",
+    tags: [ "event" ],
+    notes: "v4.5 期間限定イベントの子イベント",
```
```suggestion
    notes: "v4.5 期間限定イベント",
```

xicri commented on dataset/dictionary/items.json5:
```diff
@@ -1495,7 +1503,7 @@
     ja: "濁水幻霊イチコロリ",
     zhCN: "浊水幻灵克星",
-    pronunciationJa: "だくすいげんれいイチコロリ", // TODO Need Check
+    pronunciationJa: "だくすいげんれいイチコロリ",
```
Please revert this if there are no official sources.

```suggestion
    pronunciationJa: "だくすいげんれいイチコロリ", // TODO Need Check
```

xicri commented on dataset/dictionary/weapons.json5:
```diff
@@ -243,7 +243,7 @@
     ja: "狼牙",
     zhCN: "狼牙",
-    pronunciationJa: "ろうが", // TODO Need Check
+    pronunciationJa: "ろうが",
```
Please revert this if there are no official sources.

```suggestion
    pronunciationJa: "ろうが", // TODO Need Check
```

xicri commented on dataset/dictionary/weapons.json5:
```diff
@@ -1417,7 +1417,7 @@
     ja: "浮世の錠",
     zhCN: "尘世之锁",
-    pronunciationJa: "うきよのじょう", // TODO Need check
+    pronunciationJa: "うきよのじょう",
```
Please revert this if there are no official sources.

```suggestion
    pronunciationJa: "うきよのじょう", // TODO Need Check
```

xicri commented on dataset/dictionary/weapons.json5:
```diff
@@ -1462,7 +1462,7 @@
     ja: "碧落の瓏",
     zhCN: "碧落之珑",
-    // pronunciationJa: "へきらくのろう", // TODO Need Check
+    // pronunciationJa: "へきらくのろう",
```
Please revert this if there are no official sources.

```suggestion
    // pronunciationJa: "へきらくのろう", // TODO Need Check
```

xicri commented:
@yasuking0304 Thanks, I've merged the updates and it should be deployed in several minutes.
